docs/release-notes.md

```diff
@@ -5,22 +5,19 @@ hide:
 
 # Release Notes
 
-## Unreleased
-
-
-### Removed
-
-- `parents` attribute of MetaInfo. There are no users and uses as it only saves the name not the class itself.
+## 0.18.1
 
 ### Changed
 
 - Cleanup Model inheritance: Database related stuff is put into a mixin. The customized metaclass is moved from EdgyBaseModel to edgy.Model as well as some db related ClassVars.
```

Comment:
Don't use the word "stuff". These are release notes. People read this. Cleanup Model inheritance: Database related operations are put into a mixin. Also missing space after the `.` 

Here:

`.Model as well as some db related ClassVars.`